src/reminders.py

```diff
@@ -148,8 +151,6 @@ def parse_remind_me(time_and_reminder):
         return None, None
     if not reminder_tokens:
         return None, None
-    time_difference = time_specified - datetime.utcnow()
-    total_seconds = time_difference.total_seconds()
```

Comment:
I think these lines can stay, right? These lines are reached if we weren't able to parse a time in the `((?P<{}>\d*){})?` format, in which case we're trying to parse a time specified as a date (ie "On June 1"). All your changes should happen before the line with the `date_parse` call.

Reply:
You're correct, I must have mistakenly removed them.

Reply:
I must have removed that mistakenly, thanks for catching it.